cmd/kubectl-testkube/commands/testworkflows/run.go

```diff
@@ -69,6 +70,19 @@ func NewRunTestWorkflowCmd() *cobra.Command {
 				Config:          config,
 				DisableWebhooks: disableWebhooks,
 			})
+			if err != nil {
+				// User friendly Open Source operation error
+				errs := []error{constants.ErrOpenSourceExecuteOperationIsNotAvailable,
+					constants.ErrOpenSourceParallelOperationIsNotAvailable,
+					constants.ErrOpenSourceServicesOperationIsNotAvailable}
+				for _, e := range errs {
+					if strings.Contains(err.Error(), e.Error()) {
```

Comment:
Maybe just `Contains` on the `constants.OpenSourceOperationErrorMessage`?

Reply:
anyway, changed to parsing, should be more universal